util/CMakeLists.txt

```diff
@@ -397,6 +397,19 @@ endif()
 # Copied from: https://github.com/qedsoftware/crashpad/blob/3583c50a6575857abcf140f6ea3b8d11390205b3/util/CMakeLists.txt#L196-L233
 if(APPLE)
     if(NOT IOS)
+        if(NOT CMAKE_OSX_SYSROOT)
+            # In CMake 4.0, CMAKE_OSX_SYSROOT defaults to empty unless SDKROOT is set:
+            # https://cmake.org/cmake/help/latest/variable/CMAKE_OSX_SYSROOT.html
```

Comment:
I think it is fair to message a `WARNING` here with the text you use below for the `FATAL_ERROR`. We should use the build script to provide feedback about setting `SDKROOT` (it's like developing for Android without setting up `ANDROID_HOME`), not only if the fallback to search for the SDK path fails.